_posts/2019-10-02-#16401.md

```diff
@@ -8,7 +8,76 @@ host: jonatack
 
 ## Notes
 
+Today's PR is a Draft WIP implementation of the "package relay" concept first
+described in December 2018 in [issue
+#14895](https://github.com/bitcoin/bitcoin/issues/14895) "Package relay design
+questions" which discusses motivation and design.
+
+Package relay was proposed by [Suhas Daftuar](https://github.com/sdaftuar),
+long-time Bitcoin Core contributor and co-founder of [Chaincode
```

Comment:
I'd remove everything between the commas (`, long-time Bitcoin Core contributor and co-founder of [Chaincode Labs](https://chaincode.com)`), since it's not important for the discussion of the PR. I do think the `who [has been working](https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+author%3Asdaftuar) to improve the privacy and resilience of Bitcoin's peer-to-peer network.` is definitely useful context though.

Reply:
done, and combined the resulting first two paragraphs into one